app/src/main/java/com/asfoundation/wallet/ui/TransactionsActivity.java

```diff
@@ -582,6 +587,20 @@ private void showFingerprintTooltip(Boolean shouldShow) {
     if (shouldShow) new Handler().post(this::setTooltip);
   }
 
+  private void setupVipBadge(Menu menu) {
+    vipBadge = menu.findItem(R.id.action_vip_badge);
+    vipBadge.setVisible(false);
```

Comment:
Does this do anything?

Reply:
Yes. This is a bit of a workaround tho.
Without it, the badge shows for a brief moment before the verification. So, in the case where it's not supposed to show the badge, it appears for a split second